hack/verify-openapi-spec.sh

```diff
@@ -32,10 +32,10 @@ _tmp="${KUBE_ROOT}/_tmp"
 
 mkdir -p "${_tmp}"
 cp -a "${SPECROOT}" "${TMP_SPECROOT}"
-trap "cp -a ${TMP_SPECROOT} ${SPECROOT}/..; rm -rf ${_tmp}" EXIT SIGINT
-rm ${SPECROOT}/*
-cp ${TMP_SPECROOT}/BUILD ${SPECROOT}/BUILD
-cp ${TMP_SPECROOT}/README.md ${SPECROOT}/README.md
+trap 'cp -a ${TMP_SPECROOT} ${SPECROOT}/..; rm -rf ${_tmp}' EXIT SIGINT
+rm "${SPECROOT}/*"
```

Comment:
I don't think you want * inside quotes, see the failure

Reply:
Thanks for reminding me. You helped me a lot. You are a great person.